packages/ckeditor5-dev-release-tools/lib/utils/versions.js

```diff
@@ -41,7 +48,13 @@ export function getLastPreRelease( releaseIdentifier, cwd = process.cwd() ) {
 	return packument( packageName )
 		.then( result => {
 			const lastVersion = Object.keys( result.versions )
-				.filter( version => version.startsWith( releaseIdentifier ) )
+				.filter( version => {
+					const optionalDateIdentifier = '(-[0-9]{8})?';
+					const optionalSequenceNumber = '(.[0-9]+)?';
```

Comment:
```suggestion
					const optionalSequenceNumber = '(\.[0-9]+)?';
```

Match literal `.`

Reply:
Addressed in https://github.com/ckeditor/ckeditor5-dev/pull/1064/commits/b90cd7269511ff2273115547c3de40842cf8071d.